src/screens/Ticketing/Purchase/Overview/index.tsx

```diff
@@ -141,20 +136,6 @@ const PurchaseOverview: React.FC<OverviewProps> = ({
 
   const closeModal = () => navigation.dismiss();
 
-  const {open: openBottomSheet} = useBottomSheet();
-
-  const openTravellersSheet = () => {
-    openBottomSheet((close, focusRef) => (
-      <TravellersSheet
-        close={close}
-        save={setUserProfilesWithCount}
-        preassignedFareProduct={preassignedFareProduct}
-        userProfilesWithCount={selectableUserProfiles}
-        ref={focusRef}
-      />
-    ));
-  };
-
```

Comment:
Previously when this was a bottom-sheet you had to press a button for saving. This button was not enabled if no travellers. This validation should probably be preserved somewhat, but now for the "Til betaling" button. Talk with Hilde/Simen about how.

Reply:
Fixed in 3191f9464d16c94c9908b2d4ee708ea995bf4848.